src/offset.js

```diff
@@ -82,27 +82,28 @@ jQuery.fn.extend({
 				});
 		}
 
-		var docElem, win,
+		var docElem, win, rect,
 			elem = this[ 0 ],
-			box = { top: 0, left: 0 },
 			doc = elem && elem.ownerDocument;
 
 		if ( !doc ) {
```

Comment:
@timmywil I am not sure about this, but as I remember that test did not worker even before I made any changes. Anyway, it's what all is resolved now, thanks!